nhais/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+tornado = "*"
+pycurl = {version = "~=7.43",platform_system = "!= 'Windows'"}
```

Comment:
should we care about windows work-around?
It might be fixed anyways:
PycURL 7.43.0.5 - 2020-01-29: 'It also brings back Windows binaries'